help/view-images-and-videos.md

```diff
@@ -0,0 +1,137 @@
+# View images and videos
+
+When someone pastes or attaches an image or video to a message, Zulip shows
+a small preview. Similarly, links to images and videos will also generate a
+small preview, unless previews are [disabled](/help/allow-image-link-previews)
+in your organization. You can open the image viewer to see an image in more
+detail, download it, or watch a video.
```

Comment:
How about something like this instead?

Zulip shows previews of attached images and videos, unless previews are [disabled](/help/allow-image-link-previews) in your organization. You can click on a preview to view an image in more detail, or to play a video. Zulip also makes it convenient to browse all images and videos attached to messages in your current view.